apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -603,6 +603,17 @@ module.exports = {
                 },
               ],
             },
+            {
+              ...seoFields,
+            },
+          ],
+        },
+        {
+          label: "Contact Us | Join Us",
+          name: "contact-us-join-us",
+          file: `${APP_DIRECTORY}content/pages/contact-us-join-us.md`,
+          widget: "object",
+          fields: [
```

Comment:
Why a new page? Can't we add this `join-us` field to the _existing_ `contact-us` page?

Reply:
You [suggested](https://github.com/CodeForAfrica/ui/pull/246#discussion_r961635744) we should separate them since they are in different components.
> Looking at this, join us and contact form are two separate sections/components. Why are we combining them in the CMS?

Reply:
Yes @kelvinkipruto they **should be** separated because they're different _components_. You're separating them as if they're different _pages_.

Components in a page (Next.js) are just fields in a collection (NetlifyCMS).

Reply:
Ok. Now I get it  👨🏿‍🔧